Blockzilla/SettingsViewController.swift

```diff
@@ -152,53 +163,58 @@ class SettingsTableViewToggleCell: SettingsTableViewCell {
             let tapGesture = UITapGestureRecognizer(target: self, action: selector)
             learnMoreButton.addGestureRecognizer(tapGesture)
             addSubview(learnMoreButton)
-            
+
             // Adjust the offsets to allow the buton to fit
             spacerView.snp.makeConstraints { make in
                 make.top.bottom.leading.equalToSuperview()
                 make.trailing.equalTo(textLabel!.snp.leading)
             }
-            
+
             newLabel.snp.makeConstraints { make in
                 make.leading.equalTo(spacerView.snp.trailing)
                 make.top.equalToSuperview().offset(8)
             }
-            
+
             learnMoreButton.snp.makeConstraints { make in
                 make.leading.equalTo(spacerView.snp.trailing)
                 make.bottom.equalToSuperview().offset(4)
             }
-            
+
             newDetailLabel.snp.makeConstraints { make in
                 let lineHeight = newLabel.font.lineHeight
                 make.top.equalToSuperview().offset(10 + lineHeight)
                 make.leading.equalTo(spacerView.snp.trailing)
                 make.trailing.equalTo(contentView)
-                
+
                 if let learnMoreHeight = learnMoreButton.titleLabel?.font.lineHeight {
                     make.bottom.equalToSuperview().offset(-8 - learnMoreHeight)
                 }
             }
-            
+
             newLabel.setupShrinkage()
             newDetailLabel.setupShrinkage()
         }
     }
-    
+
+    private func setupDynamicFont() {
```

Comment:
I think you should make the version of this function that's in the superclass more generic to support the different label names. Not preferred to have two nearly identical functions when one is already inherited since ...ToggleCell inherits from the SettingsTableViewCell.